### PR TITLE
Enabling analytics benchmarking with JSON

### PR DESCRIPTION
**Pull request by GioLomia**
The purpose of this PR is to enable benchmarking similar to queries for analytics routines.

### CODE REVIEW

File: scripts/bench_python_cpp_algos.py [hunk 1]
```diff
@@ -246,6 +261,20 @@ def run_louvain(graph: Graph, input_args):
     graph.remove_node_property(property_name)
 
 
+def run_rootine(rootine, data, args_trails, argv):
```
Comment (arthurp) [inline scripts/bench_python_cpp_algos.py]: "routine"

File: scripts/bench_python_cpp_algos.py [hunk 2]
```diff
@@ -246,6 +261,20 @@ def run_louvain(graph: Graph, input_args):
...
+
+    glb_count = 0
+    for _ in range(args_trails):
+
+        start = time.time()
+        rootine(*argv)
+        data["queries"][str(rootine.__name__) + "_" + str(glb_count)] = time.time() - start
+        glb_count += 1
+
+    print(data)
```
Comment (arthurp) [inline scripts/bench_python_cpp_algos.py]: This seems like a debug print. Please remove it.

File: scripts/bench_python_cpp_algos.py [hunk 3]
```diff
@@ -28,6 +28,21 @@ def check_schema(graph: Graph, property_name):
     assert node_schema.names[new_property_id] == property_name
 
 
+def create_raw_json_dict():
```
Comment (arthurp) [inline scripts/bench_python_cpp_algos.py]: This doesn't create a json anything. What about the name `create_empty_statistics`.

**Review by arthurp — APPROVED** [review]
Please address my comments. But no need to get a rereview once you do.

**Review by gliga — APPROVED** [review]

File: scripts/bench_python_cpp_algos.py [hunk 4]
```diff
@@ -388,8 +417,49 @@ def load_graph(graph_path, edge_properties=None):
...
+
+        data = run_routine(run_tc, data, args.trials, (graph, input))
+        data = run_routine(run_cc, data, args.trials, (graph, input))
+        data = run_routine(run_kcore, data, args.trials, (graph, input))
+
+        graph_path = f"{args.input_dir}/{input['symmetric_input']}"
+        if not os.path.exists(graph_path):
+            print(f"Symmetric Graph doesn't exist: {graph_path}")
+
+        graph = load_graph(graph_path)
```
Comment (gurbinder533) [inline scripts/bench_python_cpp_algos.py:452]: Minor optimization: Louvain uses the same graph as tc, cc, and kcore with the only difference being it loads the edge property (which should not be the required thing anymore but we should test with the edge property for these graphs). So, if we can load the edge property to the existing graph, that will save some loading time for bigger graphs.

**Review by gurbinder533 — APPROVED** [review]
LGTM.

**Pull request merged**